src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionWrapper.java

```diff
@@ -0,0 +1,51 @@
+package org.jenkinsci.plugins.ewm.actions;
+
+import hudson.model.Actionable;
+import hudson.model.Run;
+import jenkins.model.RunAction2;
+import org.jenkinsci.plugins.ewm.Messages;
+
+/**
+ * A {@link RunAction2} that acts as a wrapper to hold multiple {@link ExwsAllocateActionImpl}s.
+ *
+ * @author Alexandru Somai
+ */
+public class ExwsAllocateActionWrapper extends Actionable implements RunAction2 {
```

Comment:
Ohh, Actionable action is something new :)
It may work, but I doubt it's a right approach until this wrapper provides a kind of API for accessing workspaces.
BTW I would expect this action to store a hashset (better performance than class-indexed array of ExwsAllocateActionImpl).